templates/java/build-alt.gradle.mustache

```diff
@@ -28,16 +28,37 @@ repositories {
 }
 
 dependencies {
-  compile "com.google.protobuf:protobuf-java:3.0.0-beta-2"
+  compile "com.google.protobuf:protobuf-java:3.0.0-beta-1"
   compile "io.gapi:googleapis-common-protos:0.0.0-SNAPSHOT"
+  compile "io.grpc:grpc-all:0.14.1"
```

Comment:
If you are using grpc 0.14.1, then you need to use 3.0.0-beta-2 because that's what 0.14.1 depends on:

http://mvnrepository.com/artifact/io.grpc/grpc-protobuf/0.14.1